### PR TITLE
Updated Codespaces

### PR DESCRIPTION
**Pull request by bartr**
## Type of PR

- [x] Documentation changes
- [x] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

### Purpose of PR
- updated Codespaces setup
- updated docs

### Validation
- [x] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above
 
### Issues Closed or Referenced

- References retaildevcrews/helium#396

### CODE REVIEW

File: saveenv.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+export He_Repo=helium-java
+export AUTH_TYPE=CLI
```
Comment (aflinchb) [inline saveenv.sh:4]: should be helium-csharp

File: README.md [hunk 2]
```diff
@@ -67,32 +56,34 @@ az keyvault secret show --name CosmosDatabase --vault-name $He_Name
...
+wait for `Application started. Press Ctrl+C to shut down.`
 
-# press ctl-c to stop the app
+### Testing the application
 
-```
+Open a new bash shell
 
-### In a separate terminal
+> Visual Studio Codespaces allows you to open multiple shells by clicking on the `Split Terminal` icon
```
Comment (aflinchb) [inline README.md:90]: Were we going to include a reference to httpie? Now I forget how what we decided at standup...

**Review by aflinchb — APPROVED** [review]
Approved w/ comments

File: saveenv.sh (hunk 1 above)
Comment (bartr) [reply saveenv.sh:4]: fixed

**Review by mag16 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (bartr) [reply README.md:90]: good catch - updated

**Pull request merged**